.dotnet/src/Custom/Files/FileClient.Protocol.cs

```diff
@@ -209,4 +209,36 @@ public virtual ClientResult DownloadFile(string fileId, RequestOptions options)
         using PipelineMessage message = CreateDownloadFileRequest(fileId, options);
         return ClientResult.FromResponse(_pipeline.ProcessMessage(message, options));
     }
+
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public virtual Task<ClientResult> CreateUploadJobAsync(BinaryContent content, RequestOptions options)
```

Comment:
It would be good for this to return a subclient with the "linked" protocol operations on it.